scripts/example.jl

```diff
@@ -2,26 +2,27 @@
 using FreeBird
 
 # First, we need to read the walkers from a file
+# Note that the periodic boundary conditions are updated to True, True, False
 ats = read_walkers("scripts/slab-08-4x4.extxyz",pbc="TTF")
-# Note that the periodic boundary conditions are set to True, True, False
+
+# Update the system to have 80 frozen particles and 8 free particles
```

Comment:
This is based on knowledge of the structure that was read in, correct?

Reply:
Yes, the xyz file contains the same info in this case, but it can be overwritten by the keyword argument `pbc="TTF"` if modification of the boundary conditions is needed.